packages/vm/src/buildBlock.ts

```diff
@@ -248,6 +248,7 @@ export class BlockBuilder {
     }
     let blobGasUsed = undefined
     if (tx instanceof Blob4844Tx) {
+      const { maxBlobGasPerBlock: blobGasLimit } = getBlobGasSchedule(this.vm.common)
```

Comment:
Actually I am looking through https://eips.ethereum.org/EIPS/eip-7892 for a second time and I am not finding a reference anywhere that the max blob gas per block is replaced by this new calculation (this might have been mixed up with target). 🤔  Will move back to the old version.

Reply:
I'll come back to this and do an additional review-check & cross-validate with the EIP